integration-test/src/test/java/com/datastrato/gravitino/integration/test/spark/SparkIT.java

```diff
@@ -278,6 +278,30 @@ void testAlterTableSetAndRemoveProperty() {
     Assertions.assertTrue(newProperties.containsKey("key2"));
   }
 
+  @Test
+  void testAlterTableAddAndDeleteColumn() {
+    String tableName = "test_column";
+    dropTableIfExists(tableName);
+
+    createSimpleTable(tableName);
+    List<SparkColumnInfo> sparkOldColumnInfos = getTableInfo(tableName).getColumns();
+    sparkOldColumnInfos.forEach(
+        sparkColumnInfo ->
```

Comment:
I think we should check all columns, like the origin is a,b,c,  after add column, it should be a,b,c,col1

Reply:
The default position is Last for Hive